admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -2,35 +2,43 @@
 Encryption Configuration
 ========================
 
-ownCloud includes a server-side encryption application. The Encryption app 
-encrypts all files stored on the ownCloud server, and all files on remote 
-storage that is connected to your ownCloud server. Encryption and decryption are 
-performed on the ownCloud server. All files sent to remote storage (for example 
-Dropbox and Google Drive) will be encrypted by the ownCloud server, and upon 
-retrieval, decrypted before serving them to you and anyone you have shared them 
-with.
-
-.. note:: Encrypting files increases their size by roughly 35%, so you must take 
-   this into account when you are provisioning storage and setting storage 
-   quotas. User's quotas are based on the unencrypted file size, and not the 
-   encrypted file size.
+In ownCloud 8.1 the Encryption app has been changed to the ownCloud Default 
+Encryption Module. It has a number of changes and improvements, including:
+
+* When encryption is enabled, all files are no longer encrypted at user's first 
+  logins because this caused timeouts on large installations. Instead, only 
+  files that are created or updated after encryption has been enabled are 
+  encrypted.
+
+* The  "decrypt all" option in the Personal settings has been removed, also for 
```

Comment:
At least we should clearly state that we highly recommend to make the decision (and stick to it) if you want encryption or not at the beginning when you set-up your ownCloud before you populate the instance with data and users.